backend/data/users.js

```diff
@@ -270,6 +270,14 @@ async function getCourse(id) {
   return course;
 }
 
+/**
+ * Add grade for assignment
+ * @param {string} studentId 
+ * @param {string} courseId 
+ * @param {string} assignmentId 
+ * @param {string} grade 
+ * @returns object
+ */
 async function addGrade(studentId, courseId, assignmentId, grade) {
   error.str(studentId);
```

Comment:
Technically not necessary because line 190 of assignments.js already runs error.str(studentId) before passing it into addGrade() functrion.

Reply:
I have added the changes in PR 010